### PR TITLE
Fix spank nodefile generation function

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: pbs_nodefile.c [hunk 1]
```diff
@@ -161,7 +162,7 @@ int slurm_spank_task_init(spank_t sp, int ac, char **av)
     }
     slurm_debug("spank: pbs_nodefile: SLURM_TASKS_PER_NODE: %s", taskcount);
 
-    written = snprintf(command + pos, len - pos - 1,"' SLURM_TASKS_PER_NODE='%s' %s", taskcount, generate_pbsnodefile_command);
+    written = snprintf(command + pos, len - pos - 1,"' SLURM_TASKS_PER_NODE='%s' perl -I /usr/lib64/perl5/5.32/ %s", taskcount, generate_pbsnodefile_command);
```
Comment (stdweird) [inline pbs_nodefile.c]: is this portable EL8 vs EL9?
i guess this is pointing to another issue

Comment (stdweird) [reply pbs_nodefile.c]: i suspect this can be fixed in the generate_pbsnodefile_command itself

File: generate_pbs_nodefile.pl [hunk 2]
```diff
@@ -11,6 +11,8 @@
 #   the file is printed on stdout.  Intended usage is:
 #   export PBS_NODEFILE=`generate_pbs_nodefile`
 
+BEGIN {push(@INC, '/usr/lib64/perl5/5.32/')}
```
Comment (stdweird) [inline generate_pbs_nodefile.pl:14]: test or add the el8 path as well

File: pbs_nodefile.c (hunk 1 above)
Comment (itkovian) [reply pbs_nodefile.c]: No need for el8 changes, they have a difference plugin version anyway.

Comment (stdweird) [reply pbs_nodefile.c]: not in quattor at least

Comment (stdweird) [reply pbs_nodefile.c]: imho, just add it; it has 0 risks (otoh, fixing it later is also easy)

**Pull request merged**